vault: document breaking changes to RSA key parser (#2526)
hashicorp/web-unified-docs: content/vault/v1.19.x/content/docs/updates/important-changes.mdx

#### Workaround

> Auto join will not work with this issue existing. If you need to rely on auto join for your nodes, do not upgrade.

Reviewer: ```suggestion
Auto join does not work with the affected versions. If you need to rely on auto join for your nodes, do not upgrade.
```
Style correction: avoid "this" as a pronoun, write in active voice
